GenerateToolingFeed/V3Format/V3FormatFeedEntryUpdater.cs

```diff
@@ -59,5 +59,20 @@ private static bool ShouldBeMinified(V3FormatCliEntry cliEntry)
             return os.Equals("Windows", StringComparison.OrdinalIgnoreCase)
                 && cliEntry.Architecture.Equals("x64", StringComparison.OrdinalIgnoreCase);
         }
+
+        private static string GetDownloadLink(string os, string architecture, string cliVersion, bool isMinified = false, string linkSuffix = "")
```

Comment:
I think linkSuffix would be `inproc` right?

Reply:
yeah, its passed in the caller